### PR TITLE
Refactor main method

### PR DESCRIPTION
**Pull request by mjp94**
Returning from main method after some unnecessary instructions.

### CODE REVIEW

**Comment by Scottmitch** [comment]
please fix the check style issues

```
[06:41:27][io.netty:netty-example] /var/lib/teamcity-agent/work/a00bdf77e621d005/example/src/main/java/io/netty/example/http/snoop/HttpSnoopClient.java:48: trailing whitespace
[06:41:27][io.netty:netty-example] /var/lib/teamcity-agent/work/a00bdf77e621d005/example/src/main/java/io/netty/example/http/snoop/HttpSnoopClient.java:54: trailing whitespace
```

**Comment by Scottmitch** [comment]
Can you please follow the [developer guidelines](http://netty.io/wiki/developer-guide.html) and fix the [commit message format](http://netty.io/wiki/writing-a-commit-message.html) and also sign the [ICLA](https://docs.google.com/spreadsheet/viewform?formkey=dHBjc1YzdWhsZERUQnhlSklsbG1KT1E6MQ)?

**Comment by normanmaurer** [comment]
@mjp94 please Squash the commits and adjust the message

**Comment by mjp94** [comment]
@normanmaurer how can I Squash the commits?

File: example/src/main/java/io/netty/example/http/snoop/HttpSnoopClient.java [hunk 1]
```diff
@@ -45,6 +45,11 @@
     public static void main(String[] args) throws Exception {
         URI uri = new URI(URL);
         String scheme = uri.getScheme() == null? "http" : uri.getScheme();
+        // Return if url scheme is not http(s)
+        if (!"http".equalsIgnoreCase(scheme) && !"https".equalsIgnoreCase(scheme)) {
+            System.err.println("Only HTTP and HTTP(S) is supported.");
```
Comment (johnou) [inline example/src/main/java/io/netty/example/http/snoop/HttpSnoopClient.java]: HTTP(S) means HTTP and HTTPS.

Comment (normanmaurer) [inline example/src/main/java/io/netty/example/http/snoop/HttpSnoopClient.java]: +1

Comment (mjp94) [inline example/src/main/java/io/netty/example/http/snoop/HttpSnoopClient.java]: Yes, thank you.

Comment (normanmaurer) [inline example/src/main/java/io/netty/example/http/snoop/HttpSnoopClient.java]: @mjp94 please address and squash.

**Comment by normanmaurer** [comment]
@mjp94 git rebase -i HEAD~4 and then select squash

**Comment by mjp94** [comment]
Rejecting my pull request, for some reason.

**Pull request closed without merging**